scripts/haddnano.py

```diff
@@ -57,6 +57,7 @@ def zeroFill(tree,brName,brObj) :
 				branchNames.add(br)
 				zeroFill(obj,br,otherObj.GetListOfBranches().FindObject(br))
 			#merge immediately for trees
+                elif isTree:
```

Comment:
this way it doesn't merge Events!!

Reply:
oops yes of course sorry, should be fixed now